lib/wootric/index.js

```diff
@@ -0,0 +1,91 @@
+var integration = require('analytics.js-integration');
+
+/**
+ * Expose `Wootric` integration.
+ */
+
+var Wootric = module.exports = integration('Wootric')
+  .assumesPageview()
+    .option('account_token', '')
+    .global('wootricSettings')
+    .global('wootric_survey_immediately')
+    .global('wootric')
+    //The integration object
+    .global('_wootric')
+    .tag('library', '<script src="http://d27j601g4x0gd5.cloudfront.net/segmentioSnippet.js"></script>')
+
+/**
+ * Initialize Wootric.
+ *
+ * @param {Facade} page
+ */
+
+Wootric.prototype.initialize = function(page){
+  window._wootric = this;
+  var self = this;
+  if (window.wootricSettings) {
+    window.wootricSettings.account_token = this.options.account_token;
+  } else {
+    window.wootricSettings = { account_token: this.options.account_token }
+  }
+  this.load('library', function(){
+    self.ready();
+  });
+};
+
+/**
+ * Has the Wootric library been loaded yet?
+ *
+ * @return {Boolean}
+ */
+
+Wootric.prototype.loaded = function(){
+  // We are always ready since we are just setting a global variable in initialize
+  return window.wootric;
+};
+
+/**
+ * Identify a user.
+ *
+ * @param {Facade} identify
+ */
+
+Wootric.prototype.identify = function(identify){
+  var traits = identify.traits();
+  var email = identify.email();
+  var created_at = identify.created();
+  var unixDt = Date.parse(created_at) || +created_at;
```

Comment:
Using `Date.parse` means this integration won't work with IE < 9. Does Wootric support those browsers?

Does this handle a case that `Number(new Date(createdAt))` would not?